src/App.tsx

```diff
@@ -1,14 +1,37 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
-import { TodoFilter } from './components/TodoFilter';
+import { StatusFilter, TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { useTodos } from './useTodos';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const { todos, isLoading } = useTodos();
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [titleFilter, setTitleFilter] = useState('');
+  const [statusFilter, setStatusFilter] = useState<StatusFilter>('all');
+
+  const displayTodos = useMemo(() => todos.filter(todo => {
+    if (!todo.title.toLowerCase().includes(titleFilter.toLowerCase().trim())) {
```

Comment:
This condition is really hard to read. In such cases a good practice is to first create a `const isSomething` and then put it in conditional statement.